### DISCUSSION TITLE
v2.0.0 Beta 2

### DISCUSSION THREAD

**Discussion by Kovah** [Kovah/LinkAce#841] [Announcements]
<p align="center">
<img src="https://github.com/user-attachments/assets/625aa120-8d07-4216-8741-e53915bf4ccc"/>
</p>

### This is the second BETA release of LinkAce 2

Changes from v2.0 Beta 1 to Beta 2:

- The setup now fully supports PostgreSQL and SQLite.
- Translations for DE, FR and JP were updated.
- Existing lists and tags are now properly loading on the link edit page (#835)
- The Docker image now runs rootless by default (#502)
- There is now a command to test the email configuration: `php artisan mail:check` (#500)
- A lot of CSS classes were added to all sorts of templates for better custom styling (#437)
- Readme, contribution guide and security policy were updated

<hr /><em>This discussion was created from the release <a href='https://github.com/Kovah/LinkAce/releases/tag/v2.0.0-beta2'>v2.0.0 Beta 2</a>.</em>

---

Please have a look at the [Beta 1 release notes](https://github.com/Kovah/LinkAce/discussions/832) for more details about the Beta phase of LinkAce 2.

To upgrade from Beta 1 to Beta 2, follow the standard upgrade guide for LinkAce.

**Comment by nklamann** [comment]
Is a docker setup planned ? If not, can you give a hint how to build one ?
Thanks a lot !

**Reply by Kovah** [reply]
What do you mean with Docker setup? There is a functional Docker image to run LinkAce.

**Reply by nklamann** [reply]
I mean something like the [docker setup for beta 1 ](https://github.com/Kovah/LinkAce/releases/download/v2.0.0-beta1/linkace-v2.0.0-beta1-docker.zip), but for 2.0.1 or is this just the same ?

**Reply by Kovah** [reply]
It's basically the same, but thanks for the reminder, I forgot to add the files.
